### ISSUE TITLE
each pod has only itselfs as a member 

### ISSUE THREAD

**Issue by cmeissner** [pires/hazelcast-kubernetes#17]
We using RBAC in our cluster and so i assume that the service discovery over a selector is not possible until we create a rolebinding for that. 
Any hints?

Cheers
Christian

**Comment by pires** [comment]
The error means that the app was not able to authenticate against the Kubernetes API server, maybe because there's authorization turned on and this repo doen't support it.

**Comment by cmeissner** [comment]
Yeah, I think so. But is there a plan to implement support for RBAC secured k8s clusters?

**Comment by pires** [comment]
It's been years since I last used Hazelcast, do I don't know. I believe @noctarius (Chris) had an alternative approach that may just work for you.

**Comment by noctarius** [comment]
Hazelcast has official support for kubernetes as a discovery plugin. See https://github.com/hazelcast/hazelcast-kubernetes/blob/master/README.adoc

I recommend to use the service lookup since DNS discovery is still pretty flanky.
